### DISCUSSION TITLE
List of filetypes supported?

### DISCUSSION THREAD

**Discussion by tristansly** [shibayan/WinQuickLook#108] [Q&A] [answered]
List of supported filetypes? Video?

**Comment by shibayan** [comment, marked as answer]
A list of supported file formats can be found in the implementation below.

https://github.com/shibayan/WinQuickLook/tree/master/WinQuickLook/Handlers

The supported image and video formats depend on the type of decoder installed in Windows, so it is not possible to give a list. It is environment-dependent.

**Comment by sbaum2s** [comment]
It moved to

https://github.com/shibayan/WinQuickLook/tree/master/WinQuickLook.Core/Handlers
